commons-geometry-core/src/test/java/org/apache/commons/geometry/core/precision/EpsilonDoublePrecisionContextTest.java

```diff
@@ -20,6 +20,7 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+
```

Comment:
Nit: many files have this spurious new line added before the class definition. (IDE formatter?) Obviously, this does not affect the code but let's remove them to avoid whitespace-only changes.

Reply:
my mistake. changed